api/src/main/java/org/pmiops/workbench/google/CloudStorageServiceImpl.java

```diff
@@ -30,6 +28,18 @@ public String readBlockscoreApiKey() {
     return readToString(getCredentialsBucketName(), "blockscore-api-key.txt").trim();
   }
 
+  @Override
+  public List<String> getBucketFileList(String bucketName, String directory) {
```

Comment:
opt: On my clone PR, @danqrodney requested possibly looking at size restrictions, so I might need to modify this to return `List<Blob>` instead. If you want to change now, great - otherwise I can do that in my PR.